ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -152,6 +156,32 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-single-node-workers-upgrade-conformance
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported
+  cron: 22 18 7,14,23,30 * *
```

Comment:
We will run the 5 rollback jobs in the same day?

Reply:
Yes, but at different time. We have 6 jobs on each version, each job will be run every 7 days. So almost we will have a job on each version running per day.

Reply:
In this PR, we have 5 versions, that means there will be 5 jobs running on the same day (though not on the same time), I think the quota will become tight. I suggest to generate a random cron using generate-cron-entry.sh tool. 

And is it possible to lower the frequency from f7 to f14 ?

Reply:
f14 is fine with me. Updated.

Reply:
@shellyyang1989 Is the cron items are generated by `generate-cron-entry.sh`? If not, I suggest to run `generate-cron-entry.sh <job-name>-f14 <yaml file name>` to generate a random cron item. E.g: for now, when I am searching `15,29`, I found there are two jobs are running in the same days.

Reply:
Yes,  they are. The script generated many same dates for different jobs. I manually updated some of them.

Reply:
Good to know.